src/main/java/com/google/sps/ProjectCenter.java

```diff
@@ -34,16 +38,9 @@
     GoogleCredential credential;
 
     public ProjectCenter(String projectId) throws IOException {
-        this.projectId = projectId;
-
-      // Authentication is provided by gcloud tool when running locally
-      // and by built-in service accounts when running on GAE, GCE or GKE.
-      credential = GoogleCredential.getApplicationDefault();
-      
-      // The createScopedRequired method returns true when running on GAE or a local developer
-      // machine. In that case, the desired scopes must be passed in manually. When the code is
-      // running in GCE, GKE or a Managed VM, the scopes are pulled from the GCE metadata server.
-      // See https://developers.google.com/identity/protocols/application-default-credentials for more information.
+      this.projectId = projectId;
+        
+      credential = GoogleCredential.fromStream(new FileInputStream("bt-dataflow-sql-demo.json"));
```

Comment:
Could "bt-dataflow-sql-demo.json" perhaps be passed in as a variable similar to projectId?

Reply:
# #https://github.com/googleinterns/step229-2020/pull/11/commits/74e7cee4f7154b26907db90b39558b7a9b67c0a6

Reply:
https://github.com/googleinterns/step229-2020/pull/11/commits/74e7cee4f7154b26907db90b39558b7a9b67c0a6

Reply:
Thank you for your suggestion!